tests/create-table_test.js

```diff
@@ -0,0 +1,175 @@
+require('./helper');
+
+var expect = require('chai').expect;
+var Sequelize = require('..');
+var DataTypes = Sequelize.DataTypes;
+const dialect = 'postgres';
+
+describe.only('QueryInterface', () => {
+  beforeEach(function() {
+    this.sequelize.options.quoteIdenifiers = true;
+    this.queryInterface = this.sequelize.getQueryInterface();
+  });
+
+  describe('createTable', () => {
+    // Reason: expected 'unique_rowid()' to equal 'nextval("TableWithPK_table_id_seq"::regclass)'
+    // Cockroach doesn't have nextval, only Postgres
```

Comment:
This comment is not quite right, Cockroach does have nextval, however it expects a string as the argument instead of regclass. Without the cast, this would work.

Reply:
Added a fixup to this comment.